wcfsetup/install/files/lib/form/SearchForm.class.php

```diff
@@ -564,27 +565,14 @@ protected function getConditions()
      */
     public function getUserIDs()
     {
-        $userIDs = [];
-
-        // username
-        if (!empty($this->username)) {
-            $sql = "SELECT  userID
-                    FROM    wcf" . WCF_N . "_user
-                    WHERE   username " . ($this->nameExactly ? "= ?" : "LIKE ?");
-            $statement = WCF::getDB()->prepareStatement($sql, 100);
-            $statement->execute([$this->username . (!$this->nameExactly ? '%' : '')]);
-            $userIDs = $statement->fetchAll(\PDO::FETCH_COLUMN);
-
-            if (empty($userIDs)) {
-                $this->throwNoMatchesException();
-            }
-        }
-
-        // userID
-        if ($this->userID) {
-            $userIDs[] = $this->userID;
-        }
+        return $this->userIDs;
+    }
 
-        return $userIDs;
+    /**
+     * @inheritDoc
+     */
+    public function __run()
+    {
+        throw new IllegalLinkException();
```

Comment:
Same as SearchResultPage. Especially since this Form conflicts with the new SearchPage based on the name.